src/gcode-machine-control_test.cc

```diff
@@ -112,6 +120,33 @@ class Harness {
   GCodeMachineControl *machine_control;
 };
 
+TEST(GCodeMachineControlTest, single_motor_non_euclidean_axis) {
+  struct MachineControlConfig config;
+  bzero(&config, sizeof(config));
+  config.steps_per_mm[AXIS_A] = 100;  // step/mm
+  config.acceleration[AXIS_A] = 1000;  // mm/s^2
+  config.max_feedrate[AXIS_A] = 1000;
+  HardwareMapping hardware;
+  DummyMotorOps motor_ops;
+
+  GCodeMachineControl *machine_control;
+  machine_control = GCodeMachineControl::Create(config, &motor_ops,
+                                                &hardware,
+                                                NULL,   // spindle
+                                                NULL);  // msg-stream
+  assert(machine_control != NULL);
+  // Move to pos 100, then 200, use default feedrate.
```

Comment:
There is no real default feedrate for GCode. The machine should actually report an error if you haven't given a feedrate on your first move.

The fact that BeagleG has one, is just because early testing was easier. We should report an error instead.
(or, if we want to be fancy, could have a default feedrate in the configuration, but not sure if this is not similarly obscuring the gcode-file error as it is now).